polaris-react/src/components/ShadowBevel/ShadowBevel.scss

```diff
@@ -5,6 +5,8 @@
   // stylelint-disable-next-line -- Explicitly set `0` to create a local stacking context. While the initial `z-index: auto` has a stack level of `0` within the current stacking context, it does not create a new local stacking context.
   z-index: 0;
 
+  @include scope-custom-property('shadow-bevel', 'z-index');
```

Comment:
We should never have nested ShadowBevels right? What's this for? Can you `/snapit` and test the admin thoroughly if you think we need this added scope?

Reply:
Because `<ShadowBevel>` accepts `children`, it's possible for them to be nested (even if it may not make sense from a UX perspective).

It also comes baked into `<Card>`, but that hard-codes `zIndex="32"` so wont fall afoul of the inheritance bug (the bug only occurs when there's no explicit value set - when not set, nested `<ShadowBevels>` don't pass `style={{ '--pc-shadow-bevel-z-index': zIndex }}`, so it will incorrectly inherit their `z-index` from a parent `<ShadowBevel>`).

~The final value gets applied to a `::before` pseudo element located within the `<ShadowBevel>`, which itself sets `z-index: 0` to create a new stacking context, so I _think_ this could only introduce a regression in one case: Where there's an outer `<ShadowBevel>` with a high `zIndex` prop set (say; 1000), containing a nested `<ShadowBevel>` _without_ a `zIndex` prop passed (so it incorrectly inherits the `zIndex: 1000` prop), _and_ there happens to be a child of the nested `<ShadowBevel>` (within the same stacking context) with an explicit `z-index` set which is lower than the `zIndex` prop passed to the outer `<ShadowBevel>`...~

_A quick Grokt shows no usages of `<ShadowBevel>` outside the Polaris repo_, so the above situation can never occur 🎉 

Happy to test this more thoroughly, but can you think of a way to narrow down the scope of what to test besides attempting to load every page that uses a `<Card>`? 😅